libobs/obs-output.h

```diff
@@ -33,7 +33,7 @@ extern "C" {
 #define OBS_OUTPUT_MULTI_TRACK_VIDEO (1 << 6)
 #define OBS_OUTPUT_MULTI_TRACK_AV (OBS_OUTPUT_MULTI_TRACK_AUDIO | OBS_OUTPUT_MULTI_TRACK_VIDEO)
 
-#define MAX_OUTPUT_AUDIO_ENCODERS 6
+#define MAX_OUTPUT_AUDIO_ENCODERS 32
```

Comment:
I doubt that's reasonable. I would like to see a profiling of the audio pipeline with that many tracks in a realtime recording to check that it's not detrimental latency wise. While vectorizing the tracks is definitely a good cleanup, which i myself considered, i would still stick to 6 max tracks. Power users can compile their own build with a larger number.

Reply:
I have no idea how to do profiling :(

But as I mentioned in the initial message, I've tried it with 16 tracks and it worked just fine (~5% of Ryzen 5 3600x). I insist on changing this parameter since later I would like to use its value as an upper limit for the audio mixes count input. Right now, I'm keeping `MAX_AUDIO_MIXES` at `6`, so non-power users will not be able to encode any additional tracks.

A higher number of encoding mixes might be a problem for weak machines, but it would be a very strange reason why you would limit functionality for users who have enough power to encode this.